hooks/useGovernanceAssets.ts

```diff
@@ -374,6 +374,11 @@ export default function useGovernanceAssets() {
       isVisible: canUseTransferInstruction,
       packageId: PackageEnum.Dual,
     },
+    [Instructions.DualFinanceLockupStakingOption]: {
+      name: 'GSO',
```

Comment:
Lockup Staking Option. Anywhere that is external facing this name is preferred to GSO